lib/process-services-cloud/src/lib/process/process-list/datatable/process-list-datatable-adapter.ts

```diff
@@ -21,7 +21,7 @@ import { ProcessInstanceCloudListViewModel } from '../models/perocess-instance-c
 
 export class ProcessListDatatableAdapter extends ObjectDataTableAdapter {
     constructor(data: ProcessInstanceCloudListViewModel[], schema: DataColumn<ProcessListDataColumnCustomData>[]) {
```

Comment:
@DenysVuika @eromano Do we want to provide the ability to set the `sortingMode` in these extending classes so we can utilize feature flags in the hxp-frontend-apps projects? Not sure if we want a feature flag for this but if we did, we'd need to be able to provide the sort option here.

Based on the information I've received, we don't want pages using these adapters to use client sorting so the constructor parameter would be redundant. Also, we would have to remove the constructor parameter once the feature flag is removed so it seems architecturally redundant.